src/cmd/services/m3query/config/config.go

```diff
@@ -126,9 +127,9 @@ type Configuration struct {
 	// coordinator embedded in the DB.
 	Local *LocalConfiguration `yaml:"local"`
 
-	// ClusterManagement for placemement, namespaces and database management
-	// endpoints (optional).
-	ClusterManagement *ClusterManagementConfiguration `yaml:"clusterManagement"`
+	// ClusterManagement for placement, namespaces and database management
+	// endpoints.
+	ClusterManagement ClusterManagementConfiguration `yaml:"clusterManagement"`
```

Comment:
Why change this from a pointer to directly embedded? I can't think of any reasons off the top of my head, but am wondering if there's some way this could break deserializing existing configs.

Reply:
I  changed it since I wanted to use the default placement config inside of `ClusterManagementConfiguration` directly and didn't see any issues w/ the usage of this config w/in the code. 

I don't _think_ this will affect deserialization of existing configs? It's still optional in terms of the actual underlying yaml since I moved the pointer to be the actual Etcd config so prior behaviour is retained.